itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/launch/configuration/AbstractLaunchConfigurationTest.java

```diff
@@ -37,62 +34,50 @@ public static void createNewQuarkusProject(String projectName, String projectTyp
 	}
 
 	public void testNewQuarkusConfiguration(String projectName) {
-		createNewQuarkusConfiguration(projectName);
-		checkNewQuarkusConfiguration(projectName);
-		runNewQuarkusConfiguration(projectName);
-		deleteNewQuarkusConfiguration(projectName);
+		RunConfigurationsDialog runDialog = new RunConfigurationsDialog();
+		QuarkusLaunchConfigurationTabGroup launchConfiguration = new QuarkusLaunchConfigurationTabGroup();
+		createNewQuarkusConfiguration(projectName, runDialog, launchConfiguration);
+		checkNewQuarkusConfiguration(projectName, runDialog, launchConfiguration);
+		runNewQuarkusConfiguration(projectName, runDialog, launchConfiguration);
+		deleteNewQuarkusConfiguration(projectName, runDialog, launchConfiguration);
 	}
 
-	public void createNewQuarkusConfiguration(String projectName) {
-		QuarkusLaunchConfigurationTabGroup lc = new QuarkusLaunchConfigurationTabGroup();
-		lc.selectProject(projectName);
-		lc.openRunConfiguration();
-
-		new DefaultTreeItem(TextLabels.QUARKUS_APPLICATION_TREE_ITEM).select();
-		new ContextMenuItem("New Configuration").select();
-
-		lc.setName(projectName + TextLabels.CONFIGURATION);
-
-		new PushButton("Browse...").click();
-		new DefaultTableItem(projectName).select();
-		new OkButton().click();
-
-		lc.apply();
-		new PushButton(TextLabels.CLOSE).click();
+	public void createNewQuarkusConfiguration(String projectName, RunConfigurationsDialog runDialog,
```

Comment:
you can change this to return QuarkusLaunchConfigurationTabGroup not void, so you will need to call constructor in this method.